WooCommerce/Classes/Authentication/Store Creation/Profiler/StoreCreationProfilerQuestionContainerViewModel.swift

```diff
@@ -0,0 +1,113 @@
+import Foundation
+import Yosemite
+
+enum StoreCreationProfilerQuestion: Int {
+    case sellingStatus
+    case category
+    case country
+    case challenges
+    case features
+
+    /// Progress to display for the profiler flow
+    var progress: Double {
+        switch self {
+        case .sellingStatus:
+            return 0.2
+        case .category:
+            return 0.4
+        case .country:
+            return 0.6
+        case .challenges:
+            return 0.8
+        case .features:
+            return 1
+        }
```

Comment:
Nit: To make future updates easy, we can calculate this value by using a for loop like the following. (Needs `StoreCreationProfilerQuestion` to confirm to `CaseIterable`)

```suggestion
        var progress = 0.0
        let incrementBy = 1.0 / Double(StoreCreationProfilerQuestion.allCases.count)
        for question in StoreCreationProfilerQuestion.allCases {
            progress += incrementBy

            if question == self {
                break
            }
        }
        return progress
```

Reply:
Good idea! I updated the calculation to use raw values for simplicity in 93d564e719fe431a393f1da34b37a4075c615617.